feat: charge srd for jam (#27)
fariapp/fari-games: public/documents/fari-games/charge-srd.md

# Introductions | Charge SRD

This is a condensed version of the rules of Charge RPG that you can use to make your own **Powered by Charge** games.
The Charge SRD is a condensed version of the rules of Charge RPG that you can use to make your own **Powered by Charge** games.

> This document follows a certain format that you can use as a base framework for your game.

Reviewer: Why removing "This document"?

Author: I thought "it" would be more concise, but do you think it was better the way it was before?

Reviewer: "It" would be perfect, if "This document" appeared before. With the new version of the previous line "it" refers to "The Charge SRD". It still makes sense, it's just a small shift as Charge SRD can be tought as a little more than a document, despite its name.
So both "it" and "this document" are ok, I just think the latter is better as it fits more with "follows a certain format".

Author: I'll leave it to "this document" !

> It follows a certain format that you can use as a base framework for your game, but also offers design advice so that you can hack it while still keeping the "Charge feeling"

Reviewer: Missing the final "."

You can fill-in the gaps, or tear-it down to make something completely different.

We can't wait to see what you will build with this. Enjoy!

> #### Designer Note
>

> > Text inside blockquotes is for you, the game designer.

Reviewer 2: Blockquote is kinda a markdown-centric term, is it worth clarifying "this is a blockquote" for people reading SRD?

```suggestion
> Text inside blockquotes (like this) is for you, the game designer.
```

> > Text inside blockquotes is for your, the game designer.

Reviewer: "you" or "your"?

Author: good point! Will update!

  - the GM makes a fortune roll to see what happens.
  - the player makes an action roll to see what happens.

> # Extras | Extras

Reviewer 2: Since the idea of this document is a skeleton for writing a Powered by Charge game, does it make sense to move setting and world forces (two bits that probably should always be filled out to bring the world to live) above the extras that might not even be included? 

If forces are introduced first, it makes it more straightforward to let us also flesh out the world extra more by encouraging the GM to take a moment to consider what world forces are doing.

Author: That totally makes sense to me

  - the GM makes a fortune roll to see what happens.
  - the player makes an action roll to see what happens.

# The Setting | Charge SRD

> #### Designer Note: Your Setting
>
> Use this section of the game to explain what is the setting of your game.
>
> You can talk about:
>
> - The history of the world.
> - What kind of characters the PCs are going to play.
> - The different types of problems the PCs might face. etc.

# World Forces | Charge SRD

World `forces` are independent organizations with their own unique agendas that the GM might use during the game to make it feel more vibrant. Each `force` has a specific `goal` that it will try to achieve by using `certain` methods.

> If the GM feels it makes sense, they can create a `progress clock` to represents the `force` them getting closer to their `goals`. They then mark progress on the clock when it fits the narrative - either as a `consequence` or as a result of player actions helping the `force`.

Reviewer 2: ```suggestion
If the GM feels it makes sense, they can create a `progress clock` to represent the `force` getting closer to their `goals`. They then mark progress on the clock when it fits the narrative - either as a `consequence` or as a result of player actions helping the `force`.
```

> - _Finding dirt on the government to expose corruption._
> - _Convincing the real king to take back what is theirs._
> - _Finding money to hire mercenaries to fight an eventual war._
>

> > Designing world forces is an efficient and simple way to fill the world with interesting factions that the players will be able to interact with.

Reviewer 2: This feels bare-bones and could probably use some player/GM facing notes?

```suggestion
> Designing world forces is an efficient and simple way to fill the world with interesting factions that the players will be able to interact with. 

World `forces` are independent organizations with their own agendas that the GM might use during gameplay. Different `forces` with competing goals can be used to make the world feel more vibrant as they use their own `methods` to advance their own ends. 

If the GM feels it makes sense, they can create a clock for the `force` that represents them advancing their goals, and mark progress on the clock when it suits the narrative - either as a `consequence` or as a result of player actions helping the `force`.
```

> # Attachment Extra | Extras

Reviewer 2: Does it make sense to leave a designer note here about how attachments may change the momentum dial?

> If you want to resist a `consequence`, but don’t want to make a `determination roll`, you can decide to break one of your `attachment`. Doing so cancels the consequence outright.

Reviewer 2: Should there be a bit more here to talk about how you might have attachments before jumping into how to use them? Maybe pull in first two paragraphs of the fully written extra like:

```suggestion
Characters are defined using `details` to represent who they are, what they are like and who they are linked to. By default, `details` don't have any mechanical uses in the game apart from providing narrative liberties.

An `attachment` is a character `detail` that you can choose to temporarily lose to resist an incoming `consequence`. If you want to resist a `consequence`, but don’t want to make a `determination roll`, you can decide to break one of your `attachment`. Doing so cancels the consequence outright.
```

Author: That's a very good suggestion. I love it

> However, this can only be done when it makes sense narratively, and that particular mod cannot be used until time is taken to repair or reacquire it.

Author: Bad copy and paste from LIfe Beyond Exo Station since I renamed it MODS in that game. Thanks for this

> To re-require or repair a MOD, the PC needs to start a `project progress clock` and track it on their character sheet. The clock starts at 4 segments, but may have more segments if the mod is complicated to repair, if it was heavily damaged, or if it is difficult to reacquire.

Reviewer 2: ```suggestion
To reacquire or repair an `attachment`, the PC needs to start a `project progress clock` and track it on their character sheet. The clock starts at 4 segments, but may have more segments if the mod is complicated to repair, if it was heavily damaged, or if it is difficult to reacquire.
```

> To make progress on the project, a character may only roll to repair a "broken" attachment when they have the appropriate amount of downtime.

Reviewer 2: ```suggestion
To make progress on the project, a character may only roll to repair a "broken" `attachment` when they have the appropriate amount of downtime.
```

> Once the project is completed, the character is permitted to use the attachment in the fiction, and is allowed to block a new consequence by breaking it once again.

Reviewer 2: ```suggestion
Once the project is completed, the character is permitted to use the `attachment` in the fiction, and is allowed to block a new `consequence` by breaking it once again.
```
